pkgs/tools/security/metasploit/default.nix

```diff
@@ -21,10 +21,9 @@ stdenv.mkDerivation rec {
 
     bundle install --path=$out/.gem
 
-
-    # !!! Must manually set bundler gem path !!!! 
-    export GEM_HOME=$out/.gem
-    export HOME=$out/.gem/bin:$PATH
+    # !!! This needs to be tweeked.
+    export GEM_HOME=~/gem/ruby/2.3.0/cache
```

Comment:
this is now dependent on the ruby version. look at how other packages how they are using the `ruby.gemPath`